.github/workflows/approve-snapshots.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: pnpm build
 
       - name: Serve ⚡️
-        run: pnpm serve & pnpm dlx wait-on http://localhost:3000
+        run: pnpm serve & pnpm dlx wait-on http://localhost:5000
```

Comment:
Why do we need to change the port ?

Reply:
`vite preview` starts on port 5000 -- I think port 5000 is more commonly used for this sort of thing.

Reply:
On my end, `pnpm start` still runs on `http://localhost:3000/`. That means `start` and `serve` will end up on different ports which no longer makes them interchangeable.

For example, I can no longer run the cypress tests by having the app running through `pnpm start` :confused:

Reply:
Excellent point!